versions/2.0.0/asyncapi.md

```diff
@@ -875,7 +875,7 @@ Map describing protocol-specific definitions for a server.
 Field Name | Type | Description
 ---|:---:|---
 <a name="serverBindingsObjectHTTP"></a>`http` | [HTTP Server Binding](https://github.com/asyncapi/bindings/blob/master/http#server) | Protocol-specific information for an HTTP server.
-<a name="serverBindingsObjectWebSockets"></a>`ws` | [WebSockets Server Binding](https://github.com/asyncapi/bindings/blob/master/ws#server) | Protocol-specific information for a WebSockets server.
+<a name="serverBindingsObjectWebSockets"></a>`ws` | [WebSockets Server Binding](https://github.com/asyncapi/bindings/blob/master/websockets#server) | Protocol-specific information for a WebSockets server.
```

Comment:
`ws` is correct. It's the acronym we're using on the `protocol` field too. Is there any reason behind this change?

Reply:
the link is broken:
<img width="1552" alt="Screen Shot 2019-10-22 at 2 30 26 PM" src="https://user-images.githubusercontent.com/9660342/67335504-a7fd0100-f4d8-11e9-8412-bf614fc13d05.png">

Reply:
lol, true. Thanks for fixing!